content/en/network_monitoring/performance/setup.md

```diff
@@ -87,7 +87,8 @@ Network Performance Monitoring supports use of the following provisioning system
 
 ## Setup
 
-Given this tool's focus and strength is in analyzing traffic _between_ network endpoints and mapping network dependencies, it is recommended to install it on a meaningful subset of your infrastructure and a **_minimum of 2 hosts_** to maximize value.
+Given this tool's focus and strength is in analyzing traffic _between_ network endpoints and mapping network dependencies, it is recommended to install it on a meaningful subset of your infrastructure and a **_minimum of 2 hosts_** to maximize value. 
+- The DNS monitoring feature of NPM does not need any additional work to enable it except for Kubernetes where the `collectDNSStats:` parameter needs to be set to `true` in the values.yaml file or Helm chart.
```

Comment:
This hasn't been true since helm chart 2.11.1 https://github.com/DataDog/helm-charts/pull/211/files

Reply:
Understood, I made changes and resubmitted.